template/tooling/prettier/package.json

```diff
@@ -10,6 +10,7 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
+    "prettier": "^3.0.3",
```

Comment:
This won't do much because the `@ackee/prettier-config` already includes prettier as dep.